### PR TITLE
doc(release): create v4.3.1

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5809

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 4.3.1 / 2022-07-21
```
Comment (PhilippeAccorsi) [inline changelog.md:1]:
```suggestion
# 4.3.1 / 2022-07-27
```

**Review by PhilippeAccorsi — CHANGES_REQUESTED** [review]

**Review by PhilippeAccorsi — APPROVED** [review]
Good

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# 4.3.1 / 2022-07-27
+
+### Fixed Issues
+
+- Frontend: [#5803](https://github.com/tracim/tracim/issues/5803)
+- Docker: [#5801](https://github.com/tracim/tracim/pull/5801)
+
```
Comment (gbonaspetti) [inline changelog.md:7]:
```suggestion

### Breaking/Important change
- We have updated our documentation and our scripts in order to follow  the changes in docker compose v2.

```

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+### Breaking/Important change
+- We have updated our documentation and our scripts in order to follow  the changes in docker compose v2.
```
Comment (gbonaspetti) [inline changelog.md:9]:
```suggestion
- We have updated our documentation and our scripts in order to follow [the changes in docker compose v2](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command).
```

**Pull request merged**